flink-connector-aws/flink-connector-aws-kinesis-firehose/src/main/java/org/apache/flink/connector/firehose/sink/KinesisFirehoseSinkWriter.java

```diff
@@ -220,9 +220,8 @@ private void handleFullyFailedRequest(
         }
 
         LOG.warn(
-                "KDF Sink failed to write and will retry {} entries to KDF first request was {}",
+                "KDF Sink failed to write and will retry {} entries to KDF",
```

Comment:
Can we explain here why we remove the logging for first entry in request?

Reply:
Logging the first entry in warn log here has several issues:
- records can be very big, bloating logs
- records are in serialised format (i.e. byte data), which has questionable value.
If this data is required, this can be logged separately at debug level to control log granularity